aicsimageio/aics_image.py

```diff
@@ -81,10 +81,21 @@ def __init__(self, data: typing.Union[types.FileLike, np.ndarray], **kwargs):
         Parameters
         ----------
         data: String with path to ometif/czi/tif/png/gif file, or ndarray with up to 6 dimensions
+        known_dims: Optional string with the known dimension order. If None, the reader will attempt to parse dim order.
         kwargs: Parameters to be passed through to the reader class
                        max_workers (optional Czi) specifies the number of worker threads for the backend library
         """
+
+        # Hold onto known dims until data is requested
+        if known_dims:
```

Comment:
Since `known_dims` is `None` by default anyway, can't this just be `self._known_dims`?

Reply:
Smart!